backend/canisters/local_user_index/impl/src/updates/c2c_create_user.rs

```diff
@@ -105,8 +105,8 @@ fn commit(
 
     if let Some(referred_by) = referred_by {
         if runtime_state.data.local_users.get(&referred_by).is_some() {
-            runtime_state.data.user_event_sync_queue.push(
-                referred_by.into(),
+            runtime_state.push_event_to_user(
```

Comment:
I think we can only push to local users otherwise we need to go via the user index. This is so that each canister can verify the call came from a local user index while only knowing it's own local one.

Reply:
The only difference here is that it is calling start_job_if_required after pushing to the queue

Reply:
Ahhh right yeah the user index sends a separate event to the referrer if they're on a different local user index